tests/lib/rules/jsx-curly-spacing.js

```diff
@@ -2281,5 +2281,14 @@ ruleTester.run('jsx-curly-spacing', rule, {
     }, {
       message: 'There should be no space before \'}\''
     }]
+  }, {
+    code: [
+      '<div onLayout={() => { /* dummy callback to fix android bug with component measuring */ }} />'
+    ].join('\n'),
+    output: [
+      '<div onLayout={() => { /* dummy callback to fix android bug with component measuring */ }} />'
+    ].join('\n'),
+    options: ['never', {allowMultiline: true}],
+    errors: []
```

Comment:
If this has no errors, it should be in the “valid” section, not the “invalid” section

Reply:
oh, sorry! didn't noticed that there is "valid" section

removed that test and added test into valid section